lib/pbench/cli/server/user_update.py

```diff
@@ -0,0 +1,107 @@
+import click
+from pbench import BadConfig
+from pbench.server import PbenchServerConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.common.logger import get_pbench_logger
+from pbench.server.database.models.users import User
+from pbench.server.database.database import Database
+
+_NAME_ = "pbench-user-update"
+
+
+class UserUpdate:
+    def __init__(self, context):
+        self.context = context
+
+    def execute(self):
+        config = PbenchServerConfig(self.context.config)
+
+        logger = get_pbench_logger(_NAME_, config)
+
+        # We're going to need the Postgres DB to track dataset state, so setup
+        # DB access.
+        Database.init_db(config, logger)
+
+        # Check if the user has the right privileges
+        if user is None or not user.is_admin():
```

Comment:
I think you meant to query the DB for a `user`, but that's not here as it was in delete; however I question this whole concept. The idea of simply _specifying_ a user to check admin access seems silly here; without any authorization, it simply means that you can choose to make the command succeed via `--user=admin` or choose to make it fail by specifying some other user. Let's just skip that whole game and assume that they're running on the server (which ought to be restricted) and have the right to do this. _Or_ require some real authorization. In theory, even though this is direct SQLAlchemy, we could validate an authorization token against the DB. On the other hand ... if they're running a CLI on the server, they can fire up `python3` interactively, import a few packages, and do it themselves, so the concept of authorization here is probably pointless...

Reply:
I agree, the concept of authorization here is sort of pointless and also I wasn't checking the password just that the if a specified username has `admin` role. However, if they are running CLI on the server they would already have access to DB so adding this extra layer of check would not be any useful. I will just remove this check and requirement of the `username`, we should just need a `username_to_update`